org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+  
+    <!-- Turn on the HVAC -->
+    <method name="turnOn"/>
+    
+     <!-- Turn off the HVAC -->
+    <method name="turnOff"/>
+    
+    <!-- Get the current temperature -->
+    <method name="getCurrentTemperature">
+      <arg name="temp" type="d" direction="out" />
+    </method>
+   
+   <!-- set the current temperature -->
+    <method name="setCurrentTemperature">
+      <arg name="temp" type="d" direction="in" />
+    </method>
+   
+   <!-- set the mode (e.g. Heating, Cooling, Auto)-->
+    <method name="setMode">
+      <arg name="mode" type="d" direction="in" />
```

Comment:
I would expect the mode arg type to be "s" (string). And it should be possible to get the current mode value too.

Also, I think there will need to be a way to get the set of available mode values, because different thermostats support different modes.